src/main/java/com/mraof/minestuck/player/PlayerSavedData.java

```diff
@@ -100,13 +100,17 @@ public static PlayerData getData(PlayerIdentifier player, MinecraftServer server
 	
 	public PlayerData getData(PlayerIdentifier player)
```

Comment:
I'd suggest keeping this function as it was before, and instead add a null check and null result in `PlayerSavedData.getData(ServerPlayer)`

Reply:
Gotcha